assisted_deployment.sh

```diff
@@ -273,26 +281,38 @@ function patch_extra_host_manifests() {
 }
 
 function install_assisted_service() {
- mkdir -p "${ASSETS_DIR}"
- patch_extra_host_manifests
- deploy_local_storage
- deploy_hive
- deploy_assisted_operator
+  install_prerequisites_assisted_service
+  deploy_assisted_operator
 
- oc wait -n "$ASSISTED_NAMESPACE" --for=condition=Ready pod -l app=assisted-service --timeout=90s
+  oc wait -n "$ASSISTED_NAMESPACE" --for=condition=Ready pod -l app=assisted-service --timeout=90s
+
+  echo "Installation finished..."
+  echo "For debugging purposes all the manifests have been saved in ${OCP_DIR}/saved-assets/assisted-installer-manifests"
+  echo "Please remember to manually apply BareMetalHost manifest available in the directory above."
+}
+
+# For a development workflow where we want to deploy the assisted service using operator-sdk it is
+# useful to have a process installing required dependencies, i.e. LSO and Hive as well as generating
+# AgentServiceConfig manifest that is required later on.
+function install_prerequisites_assisted_service() {
+  mkdir -p "${ASSETS_DIR}"
+  patch_extra_host_manifests
+  deploy_local_storage
+  deploy_hive
+  generate_assisted_operator
 
- echo "Installation finished..."
- echo "For debugging purposes all the manifests have been saved in ${OCP_DIR}/saved-assets/assisted-installer-manifests"
- echo "Please remember to manually apply BareMetalHost manifest available in the directory above."
+  echo "LSO and Hive have been deployed. Useful manifests are available in ${OCP_DIR}/saved-assets/assisted-installer-manifests"
```

Comment:
Better to use Local Storage Operator here

Reply:
Done